<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>release_3</class>
 <widget class="QWidget" name="release_3">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>1000</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>830</y>
     <width>111</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>发布</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>830</y>
     <width>111</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>保存</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_return">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>830</y>
     <width>111</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>返回</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>90</y>
     <width>141</width>
     <height>45</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
     <underline>false</underline>
    </font>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="text">
    <string>物品名称：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>30</y>
     <width>331</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:18pt;&quot;&gt;丢物物信息登记&lt;/span&gt;&lt;/p&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="t_name_lineEdit">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>90</y>
     <width>450</width>
     <height>45</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>180</y>
     <width>141</width>
     <height>45</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
     <underline>false</underline>
    </font>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="text">
    <string>物品类型：</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="t_class_lineEdit">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>180</y>
     <width>431</width>
     <height>45</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QComboBox" name="t_class_comboBox">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>180</y>
     <width>450</width>
     <height>45</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <item>
    <property name="text">
     <string>书籍/笔记本</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>电子产品</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>衣物</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>饰品</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>其他</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>270</y>
     <width>141</width>
     <height>45</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
     <underline>false</underline>
    </font>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="text">
    <string>丢失时间：</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lost_time_lineEdit">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>270</y>
     <width>450</width>
     <height>45</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>360</y>
     <width>141</width>
     <height>45</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
     <underline>false</underline>
    </font>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="text">
    <string>丢失地点：</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lost_addr_lineEdit_3">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>360</y>
     <width>450</width>
     <height>45</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>450</y>
     <width>141</width>
     <height>45</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
     <underline>false</underline>
    </font>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="text">
    <string>联系方式：</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>450</y>
     <width>81</width>
     <height>45</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <item>
    <property name="text">
     <string>电话</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>QQ</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>微信</string>
    </property>
   </item>
  </widget>
  <widget class="QLineEdit" name="connection_lineEdit">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>450</y>
     <width>361</width>
     <height>45</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>630</y>
     <width>171</width>
     <height>45</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
     <underline>false</underline>
    </font>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt;&quot;&gt;上传物品图片&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>540</y>
     <width>171</width>
     <height>45</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
     <underline>false</underline>
    </font>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt;&quot;&gt;物品预估价值：&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="t_value_lineEdit">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>540</y>
     <width>450</width>
     <height>45</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>1000</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="picture.qrc">:/hall_picture/640x500a0a0.jfif</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="p_lineedit">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>630</y>
     <width>450</width>
     <height>45</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
  </widget>
  <zorder>label_9</zorder>
  <zorder>t_class_comboBox</zorder>
  <zorder>pushButton</zorder>
  <zorder>pushButton_2</zorder>
  <zorder>pushButton_return</zorder>
  <zorder>label_2</zorder>
  <zorder>label</zorder>
  <zorder>t_name_lineEdit</zorder>
  <zorder>label_3</zorder>
  <zorder>t_class_lineEdit</zorder>
  <zorder>label_4</zorder>
  <zorder>lost_time_lineEdit</zorder>
  <zorder>label_5</zorder>
  <zorder>lost_addr_lineEdit_3</zorder>
  <zorder>label_6</zorder>
  <zorder>comboBox</zorder>
  <zorder>connection_lineEdit</zorder>
  <zorder>label_8</zorder>
  <zorder>label_7</zorder>
  <zorder>t_value_lineEdit</zorder>
  <zorder>p_lineedit</zorder>
 </widget>
 <resources>
  <include location="picture.qrc"/>
 </resources>
 <connections/>
</ui>
